Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -1600,9 +1600,6 @@ ROMol *reduceProductToSideChains(const ROMOL_SPTR &product,
 
             if (atommapno) {
               Atom *at = mol->getAtomWithIdx(idx);
```

Comment:
Can't at be null?  What is the guarantee here?

Reply:
`Atom::getAtomWtihIdx()` does not return nullptr. It has a `POSTCONDITION` on its result.